hr_holidays_leave_auto_approve/models/hr_leave.py

```diff
@@ -27,12 +27,14 @@ def _apply_auto_approve_policy(self):
     @api.model
     def create(self, values):
         auto_approve = self._get_auto_approve_on_creation(values)
-        tracking_disable = self.env.context.get('tracking_disable', False)
-        tracking_disable = tracking_disable or auto_approve
-        res = super(
-            HrLeave, self.with_context(
-                tracking_disable=tracking_disable)
-            ).create(values)
+        tracking_disable = self.env.context.get('tracking_disable')
```

Comment:
default value is no longer needed?

Reply:
Default value is False, that's why it's omitted here

Reply:
It seems to be that it is `None` (https://docs.python.org/3/library/stdtypes.html#dict.get)

Reply:
Ok default is `None`.
But the way standard Odoo works is without without specifying the default. Have a look in the entire standard code regarding how `'tracking_disable'` and `'mail_activity_automation_skip'` are checked.

For example you will find:


```python
        if self._context.get('tracking_disable'):
            return bla bla ..
```

or 

```python
        if self.env.context.get('mail_activity_automation_skip'):
            return bla bla ..
```

I'm going to keep this PR as it is.

Reply:
Ok, thanks for the explanation!